CONTRIBUTING.md

```diff
@@ -183,6 +183,16 @@ We like [this style guide][guide].
 
 [guide]: https://github.com/tibbe/haskell-style-guide/blob/master/haskell-style.md
 
+GitHub conventions
+-------------------
+
+Each `Cabal`/`cabal-install` release has a corresponding GitHub Project and milestone.
```

Comment:
"Each major"? 3.6.3 doesn't have a project. 3.6 does.

Reply:
Isn't 3.6.3 a minor release?

Reply:
Yeah, I meant new 3.x numbers, because that seems to be what we really plan for, while 3.x.y are reactive.

Reply:
*major are 3.x numbers